livekit-agents/livekit/agents/job.py

```diff
@@ -108,6 +108,7 @@ def __init__(
                 list[rtc.ParticipantKind.ValueType] | rtc.ParticipantKind.ValueType,
             ]
         ] = []
+        self._pending_tasks: dict[str, asyncio.Future[None]] = {}
```

Comment:
I don't think we should have a dict here, you will leak tasks if you call delete_room twice for e.g